sceptre/cli/list.py

```diff
@@ -127,3 +126,30 @@ def list_change_sets(ctx, path, url):
 
     for response in responses:
         write(response, context.output_format)
+
+
+@list_group.command(name="stacks")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def list_stacks(ctx, path):
+    """
+    List stack names and command paths for stacks in a
+    stack group.
+    \f
+
+    :param path: Path to execute the command on.
+    :type path: str
+    """
+    context = SceptreContext(
+        command_path=path,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        output_format=ctx.obj.get("output_format"),
+        options=ctx.obj.get("options"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies")
+    )
+
+    plan = SceptrePlan(context)
+    output = [f"{stack.name}.yaml: {stack.external_name}" for stack in plan.all_stacks]
```

Comment:
I'm wondering why not just make it a dictionary? It would be easier to parse that way and read even better.
```suggestion
    output = {f"{stack.name}.yaml: {stack.external_name}" for stack in plan.all_stacks}
```

Reply:
Also, I think you should have a cli flag to activate the "all_stacks" capability. When you use "all_stacks", it completely disregards the command path. 

I'm thinking the way to get all stacks would be like `sceptre list stacks --all`. That would require making the command path optional, but you could just blow up if `--all` is not specified and there's no command path.